spec/fixtures/integration/user_defined_parameterizing_rules.y

```diff
@@ -25,6 +25,7 @@ static int yyerror(YYLTYPE *loc, const char *str);
                     {
                         $$ = $1 + $2;
                         printf("(%d, %d)\n", $1, $2);
```

Comment:
[note] `$n` could be used with `skip_preprocess_references: true` because `Reference#index` is set by `#scan_reference` then no need to fill index by `#numberize_references` (`#preprocess_references`) https://github.com/ruby/lrama/blob/3b1ade0a6584db75d5800cdcd67afe7e950a4b37/lib/lrama/lexer/token/user_code.rb#L43